src/Proto.Actor/Metrics/ICountMetric.cs

```diff
@@ -0,0 +1,7 @@
+namespace Proto.Metrics
+{
+    public interface ICountMetric
+    {
+        void Inc(int count = 1, params string[]? labels);
```

Comment:
long instead of int? Prevents overflow in high-volume use cases, and will also align better with Prometheus